reya-rust-sdk/src/http_provider.rs

```diff
@@ -98,10 +90,13 @@ impl HttpProvider {
         let provider = ProviderBuilder::new()
             .with_recommended_fillers()
             .wallet(EthereumWallet::from(signer))
-            .on_http(self.url.clone());
+            .on_http(self.sdk_config.rpc_sdk_url.clone());
 
         // core create account
-        let core_proxy = CoreProxy::new(data_types::CORE_CONTRACT_ADDRESS.parse()?, provider);
+        let core_proxy = CoreProxy::new(
```

Comment:
can we check and make sure that the create account and trade flows are still working and haven't regressed?